### PR TITLE
Adds CASSANDRA_LOCAL_DC for latency-based load-balancing

### LINKED ISSUE #967: Change cassandra to use latency-aware load balancing
Per https://github.com/openzipkin/zipkin/pull/958/files#r52631161, we should consider changing the load balancing policy to be latency aware. Perhaps @yurishkuro or @danchia could shed light on how this is impactful.

The change to CassandraSpanStoreFactory would look like this:

``` scala
new TokenAwarePolicy(new LatencyAwarePolicy.Builder(new DCAwareRoundRobinPolicy(dc)).build())
```

One thing to consider is what to default `dc` to? I suppose this will end up another flag, like `CASSANDRA_LOCAL_DC`

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This allows collector and query services to avoid calling C\* hosts in
remote datacenters.

Fixes #967

### CODE REVIEW

File: zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala [hunk 1]
```diff
@@ -68,7 +67,12 @@ trait CassandraSpanStoreFactory {self: App =>
     if (cassandraUsername.isDefined && cassandraPassword.isDefined)
       builder.withCredentials(cassandraUsername(), cassandraPassword())
     builder.withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
-    builder.withLoadBalancingPolicy(new TokenAwarePolicy(new LatencyAwarePolicy.Builder(new RoundRobinPolicy()).build()))
+    builder.withLoadBalancingPolicy(new TokenAwarePolicy(
+      if (cassandraLocalDc.isDefined)
+        new LatencyAwarePolicy.Builder(DCAwareRoundRobinPolicy.builder().withLocalDc(cassandraLocalDc()).build()).build()
+      else
+        new RoundRobinPolicy()
```
Comment (yurishkuro) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala:74]: why skip latency-aware policy in this case?

Comment (codefromthecrypt) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala:74]: mistake!

Comment (codefromthecrypt) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala:74]: fixed. thanks for the catch

**Comment by yurishkuro** [comment]
lgtm

**Pull request merged**